web/templates/pages/edit_web.html

```diff
@@ -261,15 +250,12 @@
 											</div>
 										</td>
 									</tr>
+									<!-- TODO: Why is this here/how does it get shown? -->
```

Comment:
This "Home Directory" `<select>` is strange, it seems to be always hidden. Is that intentional, or a bug?

Reply:
Old vesta code *** Just remove it

Reply:
OK, I've removed this in 5fee68b96.